roomserver/roomserver.go

```diff
@@ -42,6 +42,7 @@ func SetupRoomServerComponent(
 
 	inputAPI := input.RoomserverInputAPI{
 		DB:                   roomserverDB,
+		Cfg:                  base.Cfg,
```

Comment:
We're only doing this to pull out the server name - it would be nicer if you just gave that here so we don't have to wonder what else this is doing with `Cfg`.